arch/arm/arm64/include/uk/asm/mte.h

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2021, Michalis Pappas <mpappas@fastmail.fm>.
+ * All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2022, Michalis Pappas <mpappas@fastmail.fm>.
```
I recently read an article about the `All rights reserved` clause and it basically said it does not make sense as you say with this that all rights belong to you only to say some lines below with the BSD license that you give up rights.

Reply:
Good point, I had a look at https://opensource.org/licenses/BSD-3-Clause and it's not specified there either. I also did a lookup and found some explanation in a [github issue opened in choosealicense.com](https://github.com/github/choosealicense.com/issues/556). Additional argumentation [here](https://opensource.stackexchange.com/questions/9427/with-bsd-3-do-i-need-to-propagate-an-all-rights-reserved-statement).

Other open source projects [use it nevertheless](https://fedoraproject.org/wiki/Licensing:BSD#New_BSD_(no_advertising,_3_clause)). I also did a grep in Unikraft's sources and it's pretty much everywhere 😬 

So is it safe to assume that from now on new PRs in Unikraft should not use this statement, and existing sources can left as is, given that it's a void statement anyway?

PS: I don't mind removing (or keeping) that clause, but I would rather if this PR would't block on this topic if it turns out that it will take time to come up with a final decision 🙂

Reply:
It was just a hint that this might not be necessary. I used the phrase before, too. Definitely not a blocker. You can keep it or remove it, as you like.